just_audio/example/analysis_options.yaml

```diff
@@ -0,0 +1,29 @@
+# This file configures the analyzer, which statically analyzes Dart code to
```

Comment:
This doesn't need to be part of this PR

Reply:
Sorry! It was automatically generated by flutter